msteams-platform/m365-apps/cards-loop-component.md

```diff
@@ -9,40 +9,35 @@ ms.date: 01/26/2024
-
-[Loop components](https://support.microsoft.com/office/first-things-to-know-about-loop-components-in-microsoft-teams-ee2a584b-5785-4dd6-8a2d-956131a29c81) were first released in Teams followed by other Microsoft 365 apps such as Outlook, Whiteboard, and Loop app. Loop components allow users to collaborate and share live content with others in the chat, email, meeting, Whiteboard or Loop app. Because Loop components stay in sync across different Microsoft 365 apps, they enable users to coauthor content and make real-time updates on their content. For more information, see [overview of Loop components in the Microsoft 365 ecosystem](/microsoft-365/loop/loop-components-teams).
+Adaptive Card-based Loop components enable you to build collaborative experiences within your Teams message extensions that work across Microsoft 365. Adaptive Card-based Loop components have actionable content that enables users to make live updates without having to switch contexts between Microsoft 365 apps, such as Teams and Outlook.
 
 Following is an example of an Adaptive Card-based Loop component:
 
-:::image type="content" source="~/assets/images/adaptive-cards/adaptive-card-loop.png" alt-text="Example of an Adaptive Card Loop component.":::
-
-This article provides an overview of how to build and test your Adaptive Card-based Loop components.
+:::image type="content" source="~/assets/images/adaptive-cards/adaptive-card-loop.png" alt-text="Screenshot shows an Adaptive Card-based Loop component.":::
 
-## Prerequisites
+## Build Adaptive Card-based Loop component
 
-Before you build an Adaptive Card-based Loop component, ensure that you meet the following prerequisites:
+To build an Adaptive Card-based Loop component, follow these steps:
 
 1. [Build a message extension with a search command](../messaging-extensions/what-are-messaging-extensions.md).
 
-1. Add [link unfurling](../messaging-extensions/how-to/link-unfurling.md) support to the message extension.
+1. Add [link unfurling](../messaging-extensions/how-to/link-unfurling.md) to the message extension.
 
-1. Use [Universal Actions for Adaptive Cards](../task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Work-with-Universal-Actions-for-Adaptive-Cards.md).
+1. Use [Universal Actions for Adaptive Cards](../task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Work-with-Universal-Actions-for-Adaptive-Cards.md) and define the `refresh` property to ensure that the card is always up to date. For more information, see [up-to-date cards](../task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Up-To-Date-Views.md).
 
-1. [Extend your Teams message extension across Microsoft 365](extend-m365-teams-message-extension.md).
-
-## Build Adaptive Card-based Loop component
+1. [Extend your message extension across Microsoft 365](extend-m365-teams-message-extension.md).
```

Comment:
Consider elaborating the short list of requirements to extend MEs with a sublist here:
- App manifest version 1.13 or higher
- Add Microsoft 365 channel for your bot
- Add M365 host client IDs to your app's Entra ID App Registration

Reply:
I think this is a good value addition. Done!